src/components/dls/Toast/Toast.tsx

```diff
@@ -18,6 +19,7 @@ type Action = {
 type Options = {
   preserve?: boolean;
   actions?: Action[];
+  status?: 'success' | 'error' | 'warning';
```

Comment:
Awesome! @mahmoudfelfel 

The codebase is already using enums for this use case. Might be a good idea to also use `enum` here too.

maybe we do `ToastStatus`.

other than that, looks good!.

Reply:
done, thanks for the suggestion